<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centralwidget</class>
 <widget class="QWidget" name="centralwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>791</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item>
    <widget class="QWidget" name="sideUI" native="true">
     <property name="styleSheet">
      <string notr="true">#sideUI {
       background-color: rgb(170, 0, 0);
       }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="view_btn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 15;
}
QPushButton:hover {
	color: grey;
}
#view_btn {
	font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qr_btn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
          border-radius: 15;
          }
          QPushButton:hover {
          color: grey;
          }</string>
        </property>
        <property name="text">
         <string>QR Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_btn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
          border-radius: 15;
          }
          QPushButton:hover {
          color: grey;
          }</string>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_btn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
          border-radius: 15;
          }
          QPushButton:hover {
          color: grey;
          }</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_btn">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
          border-radius: 15;
          }
          QPushButton:hover {
          color: grey;
          }</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="view_verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>View All Items:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="search_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid black;
           border-radius: 12px;
           padding-left: 5%;</string>
         </property>
         <property name="text">
          <string>Search...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <column>
        <property name="text">
         <string>Part #</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Manufacturer</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>B750 Stock</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>B757 Stock</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Minimum</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Excess</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sallies Min</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
